wgsl/index.bs

```diff
@@ -1309,57 +1309,142 @@ repeatedly replacing a subsequence with its homoglyph.)
 
 A <dfn>context-dependent name</dfn> is a [=token=] used to name a concept, but only in specific grammatical contexts.
 The spelling of the token may be the same as an [=identifier=], but the token does not [=resolve=] to a declared object.
+This section lists the tokens used as [=context-dependent names=].
+
+### Attribute Names ### {#attribute-names}
 
-Section [[#context-dependent-name-tokens]] lists all such tokens.
+The [=syntax/attribute=] names are:
```

Comment:
I like the forward links in the other subsections. Suggest adding one here.
```suggestion
See [[#attributes]].
The [=syntax/attribute=] names are:
```

Reply:
Good one, thanks!